meson.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: meson
+  version: 0.63.2
+  epoch: 0
+  description: Fast and user friendly build system
+  target-architecture:
+    - noarch
+  copyright:
+    - paths:
+        - '*'
+      attestation: TODO
+      license: Apache-2.0
+  dependencies:
+    runtime:
+      - python3
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - py3-setuptools
```

Comment:
Shouldn't `python3` be here?

Reply:
`python3` is coming in with `py3-setuptools`
```
apk add py3-setuptools
fetch https://packages.wolfi.dev/bootstrap/stage3/x86_64/APKINDEX.tar.gz
fetch https://packages.wolfi.dev/os/x86_64/APKINDEX.tar.gz
(1/13) Installing libffi (3.4.2-r1)
(2/13) Installing gdbm (1.23-r1)
(3/13) Installing xz (5.2.6-r2)
(4/13) Installing mpdecimal (2.5.1-r1)
(5/13) Installing readline (8.1.2-r1)
(6/13) Installing sqlite (3.39.2-r1)
(7/13) Installing python3 (3.10.7-r0)
(8/13) Installing py3-appdirs (1.4.4-r0)
(9/13) Installing py3-more-itertools (8.13.0-r0)
(10/13) Installing py3-ordered-set (4.0.2-r0)
(11/13) Installing py3-parsing (3.0.9-r0)
(12/13) Installing py3-packaging (21.3-r0)
(13/13) Installing py3-setuptools (59.4.0-r0)
Executing glibc-2.36-r0.trigger
Executing busybox-1.35.0-r2.trigger
OK: 2509 MiB in 53 packages
```

I wasn't sure what the right thing to do was, shall I be explicit and add `python3` here?

Reply:
Yes, I would be explicit and add python3 there, just to make it clear that this is a python-based tool.

Reply:
perfect - that makes sense